.github/actions/e2e/create-cluster/action.yaml

```diff
@@ -49,7 +49,7 @@ runs:
     run: AZURE_RESOURCE_GROUP=${{ inputs.resource_group }} AZURE_LOCATION=${{ inputs.location }} make az-create-workload-msi
   - name: create cluster
     shell: bash
-    run: AZURE_CLUSTER_NAME=${{ inputs.cluster_name }} AZURE_RESOURCE_GROUP=${{ inputs.resource_group }} AZURE_ACR_NAME=${{ inputs.acr_name }} AZURE_LOCATION=${{ inputs.location }} make az-mkaks-cilium
+    run: AZURE_CLUSTER_NAME=${{ inputs.cluster_name }} AZURE_RESOURCE_GROUP=${{ inputs.resource_group }} AZURE_ACR_NAME=${{ inputs.acr_name }} AZURE_LOCATION=${{ inputs.location }} make az-mkaks-cniv1
```

Comment:
Note this is just here so we can run the e2e tests on cniv1. I will revert this commit when we are ready to merge. For now I am leaving it so we can run our e2es against it if we need to change code in the node review

Reply:
hm I wonder if we should just run some e2e tests using both? trying to be mindful of making e2e tests take longer tho...

would we have to recreate the aks cluster to use a different CNI or can we run a patch command to update this? if so, it might be worthwhile to have at least one test that runs the different CNIs so we know if things break

Reply:
its probably better to write a specific goldpinger style suite exercising the actual node to node networking rather than just running on both. The only value of these suites running for them is that we get nodes joining apiserver and utilization tests max pods correctly.

Reply:
A matrix (ideally optional, but easy to engage on demand) would be great - and option-specific test could plug into that ...